pkg/compiler/codegen.go

```diff
@@ -1523,12 +1523,12 @@ func (c *codegen) getByteArray(expr ast.Expr) []byte {
 	}
 }
 
-func (c *codegen) convertSyscall(expr *ast.CallExpr, api, name string) {
-	syscall, ok := syscalls[api][name]
-	if !ok {
-		c.prog.Err = fmt.Errorf("unknown VM syscall api: %s.%s", api, name)
-		return
+func (c *codegen) convertSyscall(expr *ast.CallExpr) {
+	for i := len(expr.Args) - 1; i > 0; i-- {
```

Comment:
So in `f(a, b ,c)` we're calculating `c` first? It can break things.

Reply:
Not if we use it as we are currently, but fixed.